### PR TITLE
created basic database structure

### PR DESCRIPTION
**Pull request by FelixLetheren**

### CODE REVIEW

File: fish-finder_2019-08-05.sql [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+/*!40101 SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='NO_AUTO_VALUE_ON_ZERO' */;
+/*!40111 SET @OLD_SQL_NOTES=@@SQL_NOTES, SQL_NOTES=0 */;
+
+
+# Dump of table fish
+# ------------------------------------------------------------
+
+DROP TABLE IF EXISTS `fish`;
+
+CREATE TABLE `fish` (
```
Comment (MotNosniktaw) [inline fish-finder_2019-08-05.sql:28]: should name and species fields really be able to allow null?

File: fish-finder_2019-08-05.sql [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  `id` int(11) unsigned NOT NULL AUTO_INCREMENT,
+  `name` varchar(255) DEFAULT '',
+  `species` varchar(255) DEFAULT '',
+  `img-filepath` varchar(255) DEFAULT '',
+  `length` int(11) unsigned NOT NULL,
+  `agression` int(11) unsigned NOT NULL,
```
Comment (MotNosniktaw) [inline fish-finder_2019-08-05.sql]: you mentioned max values, can these be implemented in mySQL?

File: fish-finder_2019-08-05.sql [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+  `color` varchar(255) NOT NULL DEFAULT '',
```
Comment (MotNosniktaw) [inline fish-finder_2019-08-05.sql:35]: consider a look up table for color if some color values are commonly used

File: fish-finder_2019-08-05.sql (hunk 1 above)
Comment (FelixLetheren) [reply fish-finder_2019-08-05.sql:28]: Probably not! Resolved!

File: fish-finder_2019-08-05.sql (hunk 2 above)
Comment (FelixLetheren) [reply fish-finder_2019-08-05.sql]: Sort of! Resulved

File: fish-finder_2019-08-05.sql [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+DROP TABLE IF EXISTS `fish`;
+
+CREATE TABLE `fish` (
+  `id` int(11) unsigned NOT NULL AUTO_INCREMENT,
+  `name` varchar(255) NOT NULL DEFAULT '',
+  `species` varchar(255) NOT NULL DEFAULT '',
+  `img-filepath` varchar(255) DEFAULT '',
+  `length` int(11) unsigned NOT NULL,
+  `agression` enum('1','2','3','4','5') NOT NULL,
```
Comment (sjr-mayden) [inline fish-finder_2019-08-05.sql]: Aggression has two g's in it

File: fish-finder_2019-08-05.sql [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+  `color` varchar(255) NOT NULL DEFAULT '',
+  PRIMARY KEY (`id`)
+) ENGINE=InnoDB DEFAULT CHARSET=latin1;
+
+LOCK TABLES `fish` WRITE;
+/*!40000 ALTER TABLE `fish` DISABLE KEYS */;
+
+INSERT INTO `fish` (`id`, `name`, `species`, `img-filepath`, `length`, `agression`, `color`)
+VALUES
+	(2,'Grondle the Great','goldfish','/project/images/hot-grondle-pics.png',160,'5','Purple');
```
Comment (sjr-mayden) [inline fish-finder_2019-08-05.sql]: Please rename the file

File: fish-finder_2019-08-05.sql (hunk 3 above)
Comment (sjr-mayden) [reply fish-finder_2019-08-05.sql:35]: If colour of fishes starts to look repetitive I agree with Tom

File: fish-finder_2019-08-05.sql [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+DROP TABLE IF EXISTS `fish`;
+
+CREATE TABLE `fish` (
+  `id` int(11) unsigned NOT NULL AUTO_INCREMENT,
+  `name` varchar(255) NOT NULL DEFAULT '',
+  `species` varchar(255) NOT NULL DEFAULT '',
+  `img-filepath` varchar(255) DEFAULT '',
+  `length` int(11) unsigned NOT NULL,
+  `agression` enum('1','2','3','4','5') NOT NULL,
+  `color` varchar(255) NOT NULL DEFAULT '',
```
Comment (sjr-mayden) [inline fish-finder_2019-08-05.sql:35]: Pattern might be something else to look at for differentiating between fish

**Review by sjr-mayden — CHANGES_REQUESTED** [review]

**Review by sjr-mayden — APPROVED** [review]

**Pull request merged**